### PR TITLE
Add feature print format setup action

### PR DESCRIPTION
**Pull request by leonel1524**
[#118](https://github.com/adempiere/adempiere-vue/issues/118) in this PR has been added a context menu action for go directly to **Print Format Setup** window.

### CODE REVIEW

**Comment by yamelsenih** [comment]
Hi @leonel1524 can you add a gif with functionality?

Best regard

**Comment by leonel1524** [comment]
> Hi @leonel1524 can you add a gif with functionality?
> 
> Best regard

ADempiere-vue-print-format-setup

**Comment by yamelsenih** [comment]
Hi @leonel1524 maybe can be more easy if exist a stub class or static for a window inventory, some like (key, Value)

File: src/components/ADempiere/ContextMenu/contextMenuDesktop.vue [hunk 1]
```diff
@@ -35,6 +35,9 @@
               {{ $t('components.contextMenuDownload') }}
             </a>
           </el-menu-item>
+          <el-menu-item v-show="$route.name === 'Report Viewer'" index="9" @click="$router.push({ name: '8e5131dc-fb40-11e8-a479-7a0060f0aa01' })">
+            {{ $t('components.contextMenuPrintFormatSetup') }}
```
Comment (yamelsenih) [inline src/components/ADempiere/ContextMenu/contextMenuDesktop.vue:39]: Please add a Util class for launch static components from UI and add criteria for open current 
```
@AD_PrintFormat_ID@
```

**Review by yamelsenih — CHANGES_REQUESTED** [review]

File: src/components/ADempiere/ContextMenu/contextMenuDesktop.vue (hunk 1 above)
Comment (leonel1524) [reply src/components/ADempiere/ContextMenu/contextMenuDesktop.vue:39]: Updated in this [commit](https://github.com/adempiere/adempiere-vue/pull/163/commits/c997fa27a4e0ef37759b8b0dd7c6ac3cf08fa939)

**Pull request merged**